rotkehlchen/tests/api/test_custom_ethereum_tokens.py

```diff
@@ -16,6 +16,7 @@
 from rotkehlchen.tests.utils.api import (
     api_url_for,
     assert_error_response,
+    assert_proper_response,
```

Comment:
same here. Manually input ethereum token as a better name?

Reply:
Used same pattern and called it `user tokens`